internal/http/system_win.go

```diff
@@ -0,0 +1,42 @@
+// +build windows
+
+package http
+
+import (
+	"crypto/x509"
+	"syscall"
+	"unsafe"
+)
+
+func loadSystemRoots() (*x509.CertPool, error) {
```

Comment:
Doesn't `x509.SystemCertPool()` work on windows as well?

Reply:
It doesn't. See from https://github.com/golang/go/issues/16736 to a lot of discussion about it.
Using that function on windows just returns an error as you can see in [its source code](https://golang.org/src/crypto/x509/cert_pool.go?s=1328:1368#L45).

Reply:
so... I don't know if I wan't to maintain specific windows code... I haven't touch a windows machine in years and I don't have any idea about how right/secure the code you wrote is... nor will I be able to maintain it if needed :(

Reply:
Well, I seldom used windows for many years too, but I'm targeting some 50 windows developers with this changes on goreleaser.

I guess we can leave it out (avoiding the call to x509.SystemCertPool() if GOOS="windows") but that would mean those users will be second-class citizens, needing to add certificates in their's configs instead of relying on the globally installed certificates on Windows or on Windows group policy installed certificates which are managed by the corporate security and support teams.

And since the same projects will be worked on by linux and windows users that will mean that linux users will have to see/use certificates in the configuration too.

Any other ideas?